### PR TITLE
[zh-TW]: update translation for html figure

### PR DESCRIPTION
**Pull request by LinHeMa**
### Description

Update part of HTML figure zh-TW translation files.

### Motivation

I want to improve the readability of the docs.

### CODE REVIEW

File: files/zh-tw/web/html/element/figure/index.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ---
-title: <figure>：可選標題的圖片元素
+title: <figure>：非必填圖片說明的圖片元素
```
Comment (yin1999) [inline files/zh-tw/web/html/element/figure/index.md:2]: 这样翻译如何：

```suggestion
title: <figure>：具有非必填說明元素的圖片
```

File: files/zh-tw/web/html/element/figure/index.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 slug: Web/HTML/Element/figure
 ---
 
 {{HTMLSidebar}}
 
-**`<figure>`** [HTML](/zh-TW/docs/Web/HTML) 元素代表自包含內容，可能具有可選的標題，可以使用 {{HTMLElement("figcaption")}} 元素指定。該圖片、其標題和其內容被引用為一個單一單位。
+**`<figure>`** [HTML](/zh-TW/docs/Web/HTML) 元素代表自成一體的內容，可能具有非必填的圖片說明，可以使用 {{HTMLElement("figcaption")}} 元素指定。該圖片、其圖片說明和其內容被作為一個單一單位被引用。
```
Comment (yin1999) [inline files/zh-tw/web/html/element/figure/index.md:8]:
```suggestion
**`<figure>`** [HTML](/zh-TW/docs/Web/HTML) 元素代表自成一體的內容，可能具有非必填的圖片說明（使用 {{HTMLElement("figcaption")}} 元素指定）。圖片、其圖片說明和其內容被作為一個單一單位被引用。
```

File: files/zh-tw/web/html/element/figure/index.md [hunk 3]
```diff
@@ -15,9 +15,9 @@ slug: Web/HTML/Element/figure
 
 ## 使用注意事項
 
-- 通常 `<figure>` 是文件主流中引用的圖片、插圖、圖表、程式碼片段等，但可以移動到文件的其他部分或附錄，而不影響主流。
-- 可以通過在其中插入 {{HTMLElement("figcaption")}}（作為第一個或最後一個子元素）來將標題與 `<figure>` 元素連結起來。在圖片中找到的第一個 `<figcaption>` 元素被呈現為圖片的標題。
-- `<figcaption>` 為父級 `<figure>` 提供了{{glossary("accessible description", "無障礙描述")}}。
+- 通常 `<figure>` 是文件主要架構中引用的圖片、插圖、圖表、程式碼片段等，但可以移動到文件的其他部分或附錄，而不影響主要架構。
+- 可以通過在其中插入 {{HTMLElement("figcaption")}}（作為第一個或最後一個子元素）來將標題與 `<figure>` 元素產生關聯。在圖片中找到的第一個 `<figcaption>` 元素被呈現為圖片的標題。
+- `<figcaption>` 為父層 `<figure>` 提供了{{glossary("accessible description", "無障礙描述")}}。
```
Comment (yin1999) [inline files/zh-tw/web/html/element/figure/index.md:20]:
```suggestion
- `<figcaption>` 為父層 `<figure>` 提供了{{glossary("accessible name", "無障礙名稱")}}。
```

File: files/zh-tw/web/html/element/figure/index.md [hunk 4]
```diff
@@ -105,35 +105,35 @@ function NavigatorExample() {
     <tr>
```
Comment (yin1999) [inline files/zh-tw/web/html/element/figure/index.md:105]: 下面涉及的术语的翻译大多来自[內容類型](https://developer.mozilla.org/zh-TW/docs/Web/HTML/Content_categories)。不应随意修改，请更正

Comment (LinHeMa) [reply files/zh-tw/web/html/element/figure/index.md]: 感謝提醒，已修改於[b610ba2](https://github.com/mdn/translated-content/pull/25150/commits/b610ba2ca3b307c2c3662e3045a7067ec2d34445)

**Review by yin1999 — APPROVED** [review]
感谢 @LinHeMa 🙏

**Pull request merged**